frontend/src/views/user.vue

```diff
@@ -60,5 +60,5 @@ c-empty-collection(
   text-margin
 )
   c-tile-grid
-    c-user-video-preview(v-for="video of videos" :show-visibility="isPersonalPage" :video="video")
+    c-user-video-preview(v-for="video of videos" :show-visibility="isPersonalPage" :video="video" kebabMenu)
```

Comment:
We can switch to just using one variable `is-personal-page="isPersonalPage"`, then show the adorners only when it's true.